### PR TITLE
General formatting and some documentation

### PR DESCRIPTION
**Pull request by Xenmai**
Added documentation for Gamerule, Oxygen and Rotate commands.

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -828,14 +828,15 @@ public void registerCoreMembers() {
         }
 
 
-        // <--[command]
+                // <--[command]
```
Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: but y

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -2749,25 +2767,32 @@ public void registerCoreMembers() {
         // @Short Modifies the current state of a script queue.
         // @Author aufdemrand
         // @Group core
+        //
         // @Description
-        // TODO: Document Command Details
+        // TODO: Document Command Detail
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: dropped an 's' here

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -2915,36 +2950,60 @@ public void registerCoreMembers() {
...
+        // - rotate cancel <player>
         // -->
         registerCoreMember(RotateCommand.class,
                 "ROTATE", "rotate (cancel) (<entity>|...) (yaw:<#.#>) (pitch:<#.#>) (infinite/duration:<duration>) (frequency:<duration>)", 0);
 
 
         // <--[command]
         // @Name Run
-        // @Syntax run (locally) [<script>] (path:<name>) (def:<element>|...) (id:<name>) (instantly) (speed:<value>) (delay:<value>)
+        // @Syntax run (locally) [<script>] (path:<name>) (def:<element>|...) (id:<name>) (instantly) (delay:<value>)
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: hey!

Comment (BlackCoyote) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: byebye

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 4]
```diff
@@ -3299,16 +3373,21 @@ public void registerCoreMembers() {
 
         // <--[command]
         // @Name Sign
-        // @Syntax sign (type:{automatic}/sign_post/wall_sign) ["<line>|..."] [<location>] (direction:n/e/w/s)
+        // @Syntax sign (type:{sign_post}/wall_sign) ["<line>|..."] [<location>] (direction:n/e/w/s)
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: hey!

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 3 above)
Comment (Xenmai) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: check next commit, I fixed it

**Pull request merged**